tasks/parse-pdr/tests/parse_pdrs_test.js

```diff
@@ -304,19 +304,17 @@ test.serial('Empty FILE_ID valule in PDR, parse-pdr throws error', async (t) =>
 
   await s3().createBucket({ Bucket: t.context.payload.config.provider.host }).promise();
 
-  try {
-    await s3().putObject({
-      Bucket: t.context.payload.config.provider.host,
-      Key: `${t.context.payload.input.pdr.path}/${t.context.payload.input.pdr.name}`,
-      Body: streamTestData('pdrs/MOD09GQ-without-file-id-value.PDR')
-    }).promise();
-
-    await t.throws(parsePdr(t.context.payload), "Failed to parse value ('') of FILE_ID", 'Value corresponding to FILE_ID key in the PDR is empty');
-  } catch (err) {
-    if (err instanceof errors.RemoteResourceError || err.code === 'AllAccessDisabled') {
-      t.pass('ignoring this test. Test server seems to be down');
-    } else t.fail(err);
-  }
+  await s3().putObject({
```

Comment:
Is this refactor equivalent? The previous version would fail if an error was thrown by `s3().putObject()`. Or will Ava handle that automatically?

Reply:
Ava will fail the test if an error is thrown.